packages/ember-glimmer/lib/helpers/action.js

```diff
@@ -31,15 +33,21 @@ export class ClosureActionReference extends CachedReference {
     // Anything else is an action argument.
     let actionArgs = positionalValues.slice(2);
 
-    // TODO: Check if rawAction is INVOKE-able or (mut)
-
     // on-change={{action setName}}
     // element-space actions look to "controller" then target. Here we only
     // look to "target".
     let actionType = typeof rawAction;
     let action = rawAction;
 
-    if (actionType === 'string') {
+    if (isNone(rawAction)) {
+      throw new EmberError(`Action passed is null or undefined in (action) from ${target}.`);
+    } else if (rawActionRef[INVOKE]) {
+      // TODO: can we just replace this with isMut/UPDATE altogether
+      // since INVOKE is not really public API anyway?
```

Comment:
This exposes some experimentation (via private API) in addons like ember-concurrency and ember-route-action-helper.  If possible, I'd prefer to avoid breaking that...